src/open_dread_rando/files/custom_scenario.lua

```diff
@@ -476,4 +476,27 @@ end
 
 function Scenario.UpdateRoomName(new_subarea)
     RoomNameGui.Update(new_subarea)
+end
+
+function Scenario.Rando_SetAreaLocations(num_locs)
```

Comment:
drop the`Rando_` prefix on these functions pls and also rename this one to something more descriptive